apps/ledger-live-desktop/tests/specs/speculos/subAccount.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { addTmsLink } from "tests/fixtures/common";
+
+const tokens: Token[] = [
+  Token.ETH_USDT,
+  //Token.XLM_USCD, //TODO: Reactivate when Date.Parse issue is fixed - desactivate time machine for Speculos tests
+  Token.ALGO_USDT,
+  Token.TRON_USDT,
+  Token.BSC_BUSD,
+  Token.MATIC_DAI,
+];
+
+const tokensReceive: Token[] = [
+  Token.ETH_USDT,
+  Token.ETH_LIDO,
+  Token.TRON_USDT,
+  Token.TRON_BTT,
+  Token.BSC_BUSD,
+  Token.BSC_SHIBA,
+  Token.MATIC_DAI,
+  Token.MATIC_UNI,
+];
+
+const tokenERC20: Token[] = [Token.ETH_USDT];
+
+for (const [i, token] of tokens.entries()) {
+  test.describe.parallel("Add subAccount without parent @smoke", () => {
+    test.use({
+      userdata: "skip-onboarding",
+      testName: `add subAccount without parent (${token.tokenName})`,
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`Add Sub Account with parent (${token.parentAccount.currency.uiName})`, async ({
+      page,
+    }) => {
+      addTmsLink(["B2CQA-2448"]);
+
+      const app = new Application(page);
+
+      await app.portfolio.openAddAccountModal();
+      await app.addAccount.expectModalVisiblity();
+
+      await app.addAccount.selectToken(token);
+      await app.addAccount.addAccounts();
+
+      await app.addAccount.done();
+      await app.layout.goToPortfolio();
+      await app.portfolio.navigateToAsset(token.tokenName);
+      await app.account.navigateToToken(token);
+      await app.account.expectLastOperationsVisibility();
+      await app.account.expectTokenAccount(token);
+    });
+  });
+}
+
+//Reactivate test after fixing the GetAppAndVersion issue - Jira: LIVE-12581
+for (const [i, token] of tokensReceive.entries()) {
+  test.describe.skip("Add subAccount when parent exists @smoke", () => {
+    test.use({
+      userdata: "speculos-subAccount",
+      testName: `Add subAccount when parent exists (${token.tokenName})`,
+      speculosCurrency: specs[token.parentAccount.currency.deviceLabel.replace(/ /g, "_")],
+      speculosOffset: i,
+    });
+
+    test(`[${token.tokenName}] Add subAccount without parent (${token.tokenNetwork})`, async ({
+      page,
+    }) => {
+      addTmsLink(["B2CQA-640"]);
+
+      const app = new Application(page);
+
+      await app.layout.goToAccounts();
+      await app.accounts.navigateToAccountByName(token.parentAccount.accountName);
+      await app.account.expectAccountVisibility(token.parentAccount.accountName);
+
+      await app.account.clickAddToken();
+      await app.receive.selectToken(token);
+
+      await app.modal.continue();
+      await app.receive.expectValidReceiveAddress(token.parentAccount.address);
+
+      await app.speculos.expectValidReceiveAddress(token.parentAccount);
+      await app.receive.expectApproveLabel();
+    });
+  });
+}
+
+for (const [i, token] of tokens.entries()) {
+  test.describe.parallel("Token visible in parent account @smoke", () => {
```

Comment:
This can't be part of the previous test because, in the other test, I'm checking that we are on the `receive flow` for different cases (tokens with funds and tokens without funds). In the case where the token has no funds, it will not be displayed on LL.